job-dsls/jobs/srcclr_scan_pipeline.groovy

```diff
@@ -0,0 +1,65 @@
+def pipelineScript =
+        '''
+@Library('jenkins-pipeline-shared-libraries')_
+
+def repoList = []
+node('kie-rhel7') {
```

Comment:
should we add here `& !master`?

Reply:
It was already added. Please take a closer look to further commits

Reply:
label was added, but you have here a node without that.